Sources/Plasma/FeatureLib/pfConsole/pfConsoleCommands.cpp

```diff
@@ -798,14 +778,12 @@ class BriefDocGenIterator : public pfConsoleCmdIterator
         {
             fprintf(fFile, "<br />\n");
             if(depth <1)
-                strcpy(fGrpName, g->GetName());
+                fGrpName = g->GetName();
             else 
             {
                 pfConsoleCmdGroup *parentGrp;
                 parentGrp = g->GetParent();	
-                strcpy(fGrpName, parentGrp->GetName());
-                strcat(fGrpName,".");
-                strcat(fGrpName,g->GetName());
+                fGrpName = parentGrp->GetName() + '.' + g->GetName();
```

Comment:
```suggestion
                fGrpName = ST::format("{}.{}", parentGrp->GetName(), g->GetName());
```

Reply:
Out of curiosity, is this a style preference, or is `ST::format` better than plain concatenation for some other reason?

Reply:
Using `ST::format` should result in at least one less allocation here, I think. String concats are very rarely the right choice, IME, although they certainly can be the easiest.